android/library/src/main/java/io/logto/android/auth/browser/BrowserSignOutFlow.kt

```diff
@@ -77,7 +61,7 @@ class BrowserSignOutFlow(
             throw LogtoException("$SIGN_OUT_FAILED: $error")
         }
 
-        if (!uri.toString().startsWith(logtoConfig.postLogoutRedirectUri)) {
+        if (!uri.toString().startsWith(logtoAndroidClient.logtoConfig.postLogoutRedirectUri)) {
```

Comment:
since the `redirectURI` validation logic are being used multiple times, is there a way we could extract a util method?

`redirectURIValidation(uri)`